relay-server/src/services/buffer/mod.rs

```diff
@@ -466,41 +466,46 @@ mod tests {
 
     use super::*;
 
-    fn buffer_service() -> (
+    fn mock_envelope_buffer_service(
```

Comment:
It returns an actual envelope buffer service, not a mock service.

```suggestion
    fn buffer_service(
```